conanfile.py

```diff
@@ -65,6 +65,7 @@ def configure_cmake(self):
         cmake.definitions["PCRE2_BUILD_PCRE2_16"] = self.options.build_pcre2_16
         cmake.definitions["PCRE2_BUILD_PCRE2_32"] = self.options.build_pcre2_32
         cmake.definitions["PCRE2_SUPPORT_JIT"] = self.options.support_jit
+        cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
this is handled by conan automatically, isn't it?

Reply:
Yes, https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#definitions